include/os/linux/kernel/linux/blkdev_compat.h

```diff
@@ -383,7 +383,7 @@ bio_set_flush(struct bio *bio)
 static inline boolean_t
 bio_is_flush(struct bio *bio)
 {
-	return (bio_op(bio) == REQ_OP_FLUSH);
+	return (bio_op(bio) == REQ_OP_FLUSH) || (op_is_flush(bio->bi_opf));
```

Comment:
I'd put parentheses differently:
```suggestion
	return (bio_op(bio) == REQ_OP_FLUSH || op_is_flush(bio->bi_opf));
```

Reply:
ah, I thought I had copied that from the original code removed by the referenced commit, but that had on more outer pair of parentheses like this:

`return ((bio_op(bio) == REQ_OP_FLUSH) || (op_is_flush(bio->bi_opf)));`

I'm fine with all of those variants!

Reply:
Parentheses around `op_is_flush()` make no sense.  The left ones -- as you wish, but not required.  And the outer are required by style.